src/components/PurchaseSummary/PurchaseSummary.stories.tsx

```diff
@@ -0,0 +1,89 @@
+import { Meta, StoryObj } from '@storybook/react'
+import { PurchaseSummary } from './index'
+import { PurchaseSummaryProps } from './type'
+
+export default {
+  title: 'Components/PurchaseSummary',
+  component: PurchaseSummary,
+  tags: ['autodocs'],
+  argTypes: {
+    description: {
+      control: 'text',
+    },
+    totalitems: {
+      control: 'text',
+    },
+    productsPrice: {
+      control: 'number',
+    },
+    deliveryCost: {
+      control: 'number',
+    },
+    totalDiscount: {
+      control: 'number',
+    },
+    totalPrice: {
+      control: 'number',
+    },
+    installmentPayment: {
+      control: 'text',
+    },
+    discounts: {
+      control: 'object',
+    },
+  },
+  decorators: [
+    (Story) => (
+      <div style={{ padding: '16px', backgroundColor: '#f0f0f0' }}>
+        <Story />
+      </div>
+    ),
+  ],
+} as Meta<PurchaseSummaryProps>
+
+type Story = StoryObj<PurchaseSummaryProps>
+
+export const EmptyDiscounts: Story = {

```

Comment:
This name is used when displaying stories. It would be interesting to use WithDiscounts and WithoutDiscounts discounts. (or something similar)
This will be displayed as: With Discouts, Without Discouts.